.gitattributes

```diff
@@ -0,0 +1 @@
+src/generated/ diff=false
```

Comment:
Why do we need this line?

Reply:
> Why do we need this line?

protoc generated files should be treated as binary files that no need (and no way, since we can't change them by hand) for review.